.changeset/healthy-schools-promise.md

```diff
@@ -0,0 +1,19 @@
+---
+'braid-design-system': major
+---
+
+Remove legacy seekAsia themes
+
+Since the rebrand going live earlier this year, all consumers of `jobsDb`, `jobStreet`, `jobStreetClassic` and `seekUnifiedBeta` themes should now be using the `apac` theme in production.
```

Comment:
This doesn't read very smoothly to me.

```suggestion
Since the rebrand went live earlier this year, all consumers of `jobsDb`, `jobStreet`, `jobStreetClassic` and `seekUnifiedBeta` themes should now be using the `apac` theme in production.
```